packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -155,18 +178,20 @@ function EnableNotification({
           </Button>
         )}
       </div>
-      <img
-        className={classNames(
-          'absolute -bottom-2 hidden w-[7.5rem] tablet:flex',
-          acceptedJustNow ? 'right-14' : 'right-4',
-        )}
-        src={
-          acceptedJustNow
-            ? cloudinary.notifications.browser_enabled
-            : cloudinary.notifications.browser
-        }
-        alt="A sample browser notification"
-      />
+      {source !== NotificationPromptSource.SourceSubscribe && (
+        <img
+          className={classNames(
+            'absolute -bottom-2 hidden w-[7.5rem] tablet:flex',
+            acceptedJustNow ? 'right-14' : 'right-4',
+          )}
+          src={
+            acceptedJustNow
+              ? cloudinary.notifications.browser_enabled
+              : cloudinary.notifications.browser
+          }
+          alt="A sample browser notification"
+        />
+      )}
```

Comment:
Only modification I had to do is move the image since it was hidden on mobile usually. But I think its ok compromise.